examples/website/terrain/webpack.config.js

```diff
@@ -29,10 +28,7 @@ const CONFIG = {
         }
       }
     ]
-  },
-
-  // Optional: Enables reading mapbox token from environment variable
-  plugins: [new webpack.EnvironmentPlugin(['MapboxAccessToken'])]
```

Comment:
Have you tried this exampled? I think it uses a terrain layer that requires a token

Reply:
Confirmed, it doesn't work